src/DependencyInjection/KnpMenuExtension.php

```diff
@@ -75,4 +66,55 @@ public function prepend(ContainerBuilder $container): void
 
         $container->prependExtensionConfig('twig', ['paths' => [$path]]);
     }
+
+    /**
+     * @param $providers
```

Comment:
This @param without providing the type is pretty useless. Moreover, the parameter should be type-hinted, and the phpdoc should be present only in case additional information is needed (like for example for arrays)